prisma/schema.prisma

```diff
@@ -19,4 +19,18 @@ model User {
   timeAcceptPolicy String
   passwordResetToken String?
   passwordResetTokenExpires String?
+  links Link[]
+}
+
+model Link {
+  id String @id @default(auto()) @map("_id") @db.ObjectId
+  isDraft Boolean @default(true)
+  isVisible Boolean @default(true)
+  title String?
+  url String?
+  logoUrl String?
+  user User @relation(fields: [userId], references: [id])
+  userId String @db.ObjectId
+  createdAt String
+  updatedAt String
```

Comment:
Why are their types String instead of DateTime or Timestamp?

Reply:
We are storing `ISOString` instead of `UTCSeconds` here, so that's why. hahaha